src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Figures {
+    protected Name name;
+    protected Color color;
+    protected double area;
+
+    public abstract Name getName();
+
+    public abstract double getArea();
```

Comment:
getArea() method should also be provided by interface according to task
please make your Figure class implement 2 interfaces